### PR TITLE
feat: get devices by miner ID

### PR DESCRIPTION
**Pull request by oopcode**

### CODE REVIEW

File: insonmnia/hub/server.go [hunk 1]
```diff
@@ -650,6 +628,29 @@ func (h *Hub) Devices(ctx context.Context, request *pb.Empty) (*pb.DevicesReply,
 	return reply, nil
 }
 
+func (h *Hub) MinerDevices(ctx context.Context, request *pb.ID) (*pb.DevicesReply, error) {
+	h.mu.Lock()
+	defer h.mu.Unlock()
```
Comment (antmat) [inline insonmnia/hub/server.go]: Looks like this lock is held a bit longer than needed.
Looks like h.getMinerByID fits here

Comment (oopcode) [reply insonmnia/hub/server.go]: Fixed.

**Comment by antmat** [comment]
Let's keep commit history clean. It' better to rebase on master to avoid merge commits.

**Comment by Sokel** [comment]
uses merging instead of rebase due the merge conflicts, not did all.

**Pull request merged**